### PR TITLE
Added ci-release integragion

### PR DESCRIPTION
**Pull request by barambani**
Closes https://github.com/laserdisc-io/laserdisc/issues/11

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -231,11 +236,9 @@ lazy val publishSettings = Seq(
       "scm:git:git@github.com:laserdisc-io/laserdisc.git"
     )
   ),
-  homepage := Some(url("http://laserdisc.io")),
   licenses := Seq("MIT" -> url("https://raw.githubusercontent.com/laserdisc-io/laserdisc/master/LICENSE")),
   pgpPublicRing := file(".travis/local.pubring.asc"),
-  pgpSecretRing := file(".travis/local.secring.asc"),
-  releaseEarlyWith := SonatypePublisher
+  pgpSecretRing := file(".travis/local.secring.asc")
```
Comment (sirocchj) [inline build.sbt:241]: This line we shouldn’t need anymore, same for the file `.travis/secrets.tar.enc`

File: .travis.yml [hunk 2]
```diff
@@ -1,50 +1,101 @@
...
 language: scala
+
+before_install:
+- git fetch --tags
+
+before_cache:
+- find $HOME/.ivy2/cache -name "ivydata-*.properties" | xargs rm
+- find $HOME/.ivy2/cache -name "*SNAPSHOT*" | xargs rm -rf
+- find $HOME/.ivy2/cache -name "*.lock" | xargs rm -rf
+- find $HOME/.coursier/cache -name "*.lock" -type f -delete
```
Comment (sirocchj) [inline .travis.yml:12]: Not sure there’s a lock there but worth checking

File: .travis.yml [hunk 3]
```diff
@@ -1,50 +1,101 @@
...
+- name: release-212
+  if: branch = master AND tag IS present AND NOT fork
+- name: release-213
+  if: branch = master AND tag IS present AND NOT fork
+- name: cleanup
+  if: tag IS NOT present
+
 scala:
-- 2.12.11
+- 2.12.12
```
Comment (sirocchj) [inline .travis.yml:34]: This version I don’t think is out yet?

File: .travis.yml [hunk 4]
```diff
@@ -1,50 +1,101 @@
...
-  fi
-- bash <(curl -s https://codecov.io/bash)
+services:
+- docker
 
-after_script:
-- docker stop $(docker ps -a -q)
+cache:
+  directories:
+  - "$HOME/.coursier"
```
Comment (sirocchj) [inline .travis.yml:95]: The cache dir is different now see https://get-coursier.io/docs/cache.html#default-location

**Comment by barambani** [comment]
good to go. Will see if it releases.

File: build.sbt [hunk 5]
```diff
@@ -231,11 +231,8 @@ lazy val publishSettings = Seq(
       "scm:git:git@github.com:laserdisc-io/laserdisc.git"
     )
   ),
-  homepage := Some(url("http://laserdisc.io")),
   licenses := Seq("MIT" -> url("https://raw.githubusercontent.com/laserdisc-io/laserdisc/master/LICENSE")),
-  pgpPublicRing := file(".travis/local.pubring.asc"),
-  pgpSecretRing := file(".travis/local.secring.asc"),
-  releaseEarlyWith := SonatypePublisher
+  pgpPublicRing := file(".travis/local.pubring.asc")
```
Comment (sirocchj) [inline build.sbt:235]: I don’t think we need this either, do we?

Comment (barambani) [reply build.sbt:235]: ops :)

**Pull request merged**